src/main/data/DataStorage.txt

```diff
@@ -0,0 +1,6 @@
+T | 1 | read book
+D | 0 | return book | June 6th
+E | 1 | project meeting | Aug 6th 2-4pm
+E | 0 | badminton | 10pm
+T | 1 | eat dinner
+D | 0 | eat meal | 12pm
```

Comment:
Perhaps you can consider adding this file into .gitignore since this is a personalised information for a user.